Dockerfile.local

```diff
@@ -1,4 +1,4 @@
-ARG GOLANG=golang:1.22.5-alpine3.19
+ARG GOLANG=golang:1.22.6-alpine3.19
```

Comment:
This is the only alpine image not in 3.20 version besides the go.mod and go.sum files missing